python-pytest.el

```diff
@@ -125,6 +125,7 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
    [("-c" "color" "--color")
     ("-q" "quiet" "--quiet")
     ("-s" "no output capture" "--capture=no")
+    ("-w" "disable warnings" "-W ignore")
```

Comment:
let's keep it `-W` (capital) so that it matches both the corresponding and `pytest` and `python` args (both also use `-W`)

Reply:
```suggestion
    ("-W" "ignore warnings" "-W ignore")
```